### PR TITLE
Add a format action based on comment

### PR DESCRIPTION
**Pull request by balghane**
This removes the auto-formatting that was happening before.

### CODE REVIEW

File: .github/workflows/comment_format.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            const {owner, repo} = context.issue
+            github.reactions.createForIssueComment({
+              owner,
+              repo,
+              comment_id: context.payload.comment.id,
+              content: "rocket",
+            });
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ ssecrets.FORMATTER_PAT }}
```
Comment (tails618) [inline .github/workflows/comment_format.yml]: Should be `secrets`, not `ssecrets`.

File: .github/workflows/comment_format.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Fetch all history and metadata
+        run: |
+          git fetch --prune --unshallow
+          git checkout -b pr
+          git branch -f main origin/main
+      - name: Checkout PR
+        run: |
+          gh pr checkout $NUMBER
+        env:
+          GITHUB_TOKEN: "${{ ssecrets.FORMATTER_PAT }}"
```
Comment (tails618) [inline .github/workflows/comment_format.yml]: `ssecrets`

**Review by tails618 — CHANGES_REQUESTED** [review]

File: .github/workflows/comment_format.yml (hunk 1 above)
Comment (balghane) [reply .github/workflows/comment_format.yml]: fixed

File: .github/workflows/comment_format.yml (hunk 2 above)
Comment (balghane) [reply .github/workflows/comment_format.yml]: fixed

File: .github/workflows/comment_format.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+name: Format PR on comment
```
Comment (avakimc) [inline .github/workflows/comment_format.yml:1]: Do I have to understand this/What is all this?

File: build.gradle [hunk 4]
```diff
@@ -1,6 +1,17 @@
 plugins {
     id "java"
     id "edu.wpi.first.GradleRIO" version "2023.1.1"
+    id 'com.diffplug.spotless' version '6.13.0'
+}
```
Comment (avakimc) [inline build.gradle:5]: What does this do?

**Review by avakimc — APPROVED** [review]

**Review by tails618 — APPROVED** [review]

**Pull request merged**